packages/Search/src/details/DTK_DetailsAlgorithms.hpp

```diff
@@ -192,6 +195,18 @@ Point return_centroid( Box const &box )
 KOKKOS_INLINE_FUNCTION
 Point return_centroid( Sphere const &sphere ) { return sphere.centroid(); }
 
+// transformation that maps the unit cube into a new axis-aligned box
+KOKKOS_INLINE_FUNCTION
+void translateAndScale( Point const &in, Point &out, Box const &ref )
```

Comment:
The name does not follow other names in file. Should this be `translate_and_scale`? And the order of arguments is weird, mixing in and out.

Reply:
Also, please add a comment that it can be done in-place (i.e., `in` and `out` may be the same object).

Reply:
> name does not follow other names in file

Will change `return_centroid()` to `returnCentroid()`.  That's the coding convention in DTK.

> comment that it can be done in-place

Good point

Reply:
> Do you have a strong opinion on this?

No.